airbyte-cdk/python/airbyte_cdk/logger.py

```diff
@@ -60,7 +60,9 @@ class AirbyteLogFormatter(logging.Formatter):
 
     def format(self, record: logging.LogRecord) -> str:
         """Return a JSON representation of the log message"""
-        airbyte_level = self.level_mapping.get(record.levelno, "INFO")
+        airbyte_level = self.level_mapping.get(record.levelno, Level.INFO)
+        if "as_is" in record.__dict__.keys():
+            return record.msg
```

Comment:
Total nit, but if we're aborting, we can swap the order of these and skip the level remap logic when it is already formatted.

```suggestion
        if "as_is" in record.__dict__.keys():
            # Preformatted. Emit as-is.
            return record.msg

        airbyte_level = self.level_mapping.get(record.levelno, Level.INFO)
```